site/docs/v1/tech/identity-providers/epic-games.adoc

```diff
@@ -0,0 +1,57 @@
+---
+layout: doc
+title: Epic Games Identity Provider
+description: Login with Epic Games using the Epic Games Identity Provider
+---
+
+:sectnumlevels: 0
+
+[NOTE.since]
+====
+As of version 1.28.0, this IdP is available.
+====
+
+== Overview
+
+This section will cover how to add a Login with Epic Games button to FusionAuth. However, this section will not cover how to obtain programmatic access to Epic Games IdP itself.
```

Comment:
Oh, my intention was to indicate that we would not be doing a screenshot walk-through of how to sign up for certain IdP due to the complexity of the initial setup. :)  I might remove the reference; it may not be needed.